ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -20,15 +20,36 @@ namespace irr { namespace ext { namespace MitsubaLoader {
    from CMitsubaLoader::loadAsset, and in other situations only warning should be shown.)
  - handle 'version' attribute
 
- - make sure default values for <float .. /> <boolean .. /> etc. are correct
- - test atof function (CSimpleElement.h)
+ - make sure that default values for <float .. /> <boolean .. /> etc. are correct
+ - test atof function (CSimpleElement.h) or use ostringstream operator >> instead
  
  - how should loader treat upper/lower case letters in xml?
 
  - clean up memory dynamically allocated by ParserData::scene
 
  - resolve header include conflict (IElement.h needs to include ParserUtil.h (only for ParserLog), and ParserUtil.h needs to include IElement.h)
 
+ - check, if transform name is indeed "toWorld"
+
+ - create an issue about isALoadableFileFormat in CBAWMeshFileLoader
+
+ - disable asset cacheing (use ECF_DUPLICATE_TOP_LEVEL)
```

Comment:
why?

Reply:
needed that for "baking" mesh transforms, but it is irrelevant now